Storage/src/Connection/Rest.php

```diff
@@ -600,4 +602,26 @@ protected function supportsBuiltinCrc32c()
     {
         return Builtin::supports(CRC32::CASTAGNOLI);
     }
+
+    /**
+     * Add the required retry function and send the request.
+     *
+     * @param string $resource resource name, eg: buckets.
+     * @param string $method method name, eg: get
+     * @param array $args
+     */
+    private function sendRequest($resource, $method, $args)
+    {
+        $args['restRetryFunction'] = $this->getRestRetryFunction($resource, $method, $args);
+        if ($resource === 'bucket_acl') {
```

Comment:
Doesn't `$args['type']` contain the string `bucket_acl` ?

Reply:
the type they send is
'bucketAccessControls',
'defaultObjectAccessControls',
'objectAccessControls'

Ref: https://github.com/googleapis/google-cloud-php/blob/main/Storage/src/Acl.php#L66